sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -2512,12 +2519,12 @@ class Dataset[T] private[sql](
   /**
    * Computes specified statistics for numeric and string columns. Available statistics are:
    *
-   * - count
-   * - mean
-   * - stddev
-   * - min
-   * - max
-   * - arbitrary approximate percentiles specified as a percentage (eg, 75%)
+   *  - count
+   *  - mean
+   *  - stddev
+   *  - min
+   *  - max
+   *  - arbitrary approximate percentiles specified as a percentage (eg, 75%)
```

Comment:
ditto about listing up. This listing up doesn't work already:
![Screen Shot 2019-12-17 at 8 47 13 PM](https://user-images.githubusercontent.com/6477701/70993036-70908680-210e-11ea-840b-cc0d7a25fe56.png)

Reply:
![image](https://user-images.githubusercontent.com/8326978/70993289-a92c6200-2106-11ea-8024-fb9f33be3fa0.png)
one more space is enough for listing up

Reply:
Yeah, but it doesn't work in javadoc. I meant this is right fix for scaladoc but not for javadoc.

Reply:
oh I missed java doc